deploy/iso/Dockerfile

```diff
@@ -17,7 +17,7 @@
 FROM boot2docker/boot2docker:1.12.3
 
 # Set the version of Docker and the expected sha.
-RUN echo "1.11.2" > $ROOTFS/etc/version
+RUN echo "1.11.3" > $ROOTFS/etc/version
```

Comment:
@budhrg shouldn't it be 1.12.3 instead of 1.11.3? Also can you please open an issue for it?